e2e/pages/filing-app/formAlerts.spec.ts

```diff
@@ -0,0 +1,259 @@
+import { expect } from '@playwright/test';
+import path from 'node:path';
+import { test } from '../../fixtures/testFixture';
+
+test('Form Alerts', async ({ page }) => {
+  test.slow();
+
+  // Filing page
+  await test.step('Filing page', async () => {
+    await page.goto('/filing');
+    await expect(page.locator('h1')).toContainText(
+      'File your small business lending data',
+    );
+    await expect(
+      page.getByRole('button', { name: 'Start filing' }),
+    ).toBeVisible();
+    await page.getByRole('button', { name: 'Start filing' }).click();
```

Comment:
Let's talk about making our fixtures work better, since we should be able to use `navigateToProvideTypeOfFinancialInstitution`, but I think we discussed that you felt it was repetitive to go through the previous steps just to get to this page. Totally understandable, but let's chat about how to make it better for a bit at our meeting later.